packages/application-shell/src/components/application-shell/application-shell.tsx

```diff
@@ -534,13 +536,8 @@ const ApplicationShell = <AdditionalEnvironmentProperties extends {}>(
 
   return (
     <>
-      <Global
-        styles={css`
-          #app {
-            height: 100%;
-          }
-        `}
-      />
+      <ThemeProvider theme="default" />
+      <GlobalStyles />
```

Comment:
This is an unrelated proposed change: I notices we had global styles in this component and also in the `ApplicationShellProvider`. I wasn't expecting styling in that component but only context data related logic because its a provider.
Since we already had one global styling in this component (`ApplicationShell`) I thought it could make sense to unify them and use them here.